README.md

```diff
@@ -9,10 +9,48 @@ This project is based on Queso Queue Plus, which was originally developed by Sho
 
 ## How do I use this?
 
+### Docker instructions
+
+You need to have docker engine installed on the system for this to work.  
+
 First, you must clone the project. Either download the source or run the following command:
 
 ```bash
-  git clone https://github.com/fluid-queue/fluid-queue
+  git clone https://github.com/fluid-queue/fluid-queue.git
+```
+
+
+After that, copy (detailed below) and configure the `settings.json` file with your favorite text editor. 
+```bash
+cp settings.example.json settings.json
+
+# now edit settings.json with your favorite text editor
+```
+
+Next, build and run the image (detached):
+
+```bash
+  docker compose up -d 
+```
+
+To close the queue press `CTRL + C` inside the terminal.
+```bash
+  docker compose stop 
+```
+
+The container will restart unless stopped, including through a reboot. The queue will be persisted on your local host in the data folder - `data/queue.json` & `data/customCodes.json`.
```

Comment:
The `./customCodes.json` has not moved to `./data/customCodes.json` yet.
See #10.